content/automate/ManageIQ/System/CommonMethods/QuotaStateMachine.class/__methods__/rejected.rb

```diff
@@ -1,9 +1,39 @@
+# frozen_string_literal: true
+
 #
 # Description: Quota Exceeded rejected method.
 #
+module ManageIQ
+  module Automate
+    module System
+      module CommonMethods
+        module QuotaStateMachine
+          class Rejected
+            def initialize(handle = $evm)
+              @handle = handle
+            end
+
+            def main
+              rejected
+            end
+
+            private
+
+            def miq_request
+              @handle.root['miq_request']
+            end
+
+            def rejected
+              @handle.log('info', "Request denied because of #{miq_request.message}")
```

Comment:
Not sure the `miq_request` method is buying much here, and in fact it is slightly inefficient compared to the previous logic since it is causing the code to do the `@handle.root['miq_request']` lookup every time it is referenced instead of just once.

I would just keep the previous logic unless this is useful for testing, in which the method should cache the result on the first call and return it for other callers.